src/main/webapp/app/app.tsx

```diff
@@ -58,7 +59,7 @@ const App: React.FunctionComponent<IAppProps> = (props: IAppProps) => {
 const mapStoreToProps = ({ authStore, layoutStore, firebaseAppStore }: IRootStore) => ({
   isAuthorized: authStore.isAuthorized,
   authorities: authStore.account.authorities,
-  isCurator: hasAnyAuthority(authStore.account.authorities, [AUTHORITIES.CURATOR]),
+  isCurator: hasAnyAuthority(authStore.account.authorities ?? [], [AUTHORITIES.CURATOR]),
```

Comment:
I use `??` since it's a check for `null` or `undefined` instead of doing a "falsy" because that's normally what you want anyway. So for example you could have a check on a number like this `0 || 1` which will result in the value being 1. With `0 ?? 1` the result is 0.